### ISSUE TITLE
Tuntikirjanpidolle template

### ISSUE THREAD

**Issue by LasAtt** [javaLabra/Javalabra2016-3#1]
Meidän kannattaisi tarjota tuntikirjanpidolle markdown-template, nykyisellään kaikissa projekteissa tuntikirjanpidot on tyylitelty eri tavalla.

**Comment by Ooppa** [comment]
## Esimerkiksi muotoa:

### Tuntikirjanpito

| Päivä | Tunnit | Kuvaus |
| --- | --- | --- |
| 01.01.2016 | 1.5h | Aiheen suunnittelua ja repon luonti |
| ... | ... | ... |

**Comment by mluukkai** [comment]
Joo, tuo on hyvä idea. Jos formaatti olisi määrämuotoinen, voitaisiin
tarvittaessa generoida jollain skriptillä statistiikkaa käytetyistä
työtunneista.
1. tammikuuta 2016 klo 1.19 Ooppa notifications@github.com kirjoitti:
